public/apps/configuration/panels/role-list.tsx

```diff
@@ -110,6 +111,55 @@ const columns = [
   },
 ];
 
+/* 
```

Comment:
it is good to see this example input and output, since we already have example data here. would it possible to move this transfer to a separate utils file, it will be much easy to unit test.